modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp

```diff
@@ -1930,5 +1930,186 @@ PERF_TEST_P_(ResizeFxFyPerfTest, TestPerformance)
 
 //------------------------------------------------------------------------------
 
+PERF_TEST_P_(ParseSSDWLPerfTest, TestPerformance)
+{
+    ParsersTest parser;
```

Comment:
Try to make `ParseSSDWLPerfTest` heir to `ParsersTests`

Reply:
Great catch. Fixed.